frameworks/Java/servicetalk/src/main/java/servicetalk/Server.java

```diff
@@ -0,0 +1,100 @@
+package servicetalk;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+import io.servicetalk.concurrent.api.AsyncContext;
+import io.servicetalk.concurrent.api.Completable;
+import io.servicetalk.http.api.*;
+import io.servicetalk.http.netty.HttpProtocolConfigs;
+import io.servicetalk.http.netty.HttpServers;
+import io.servicetalk.data.jackson.JacksonSerializationProvider;
+import io.servicetalk.transport.api.ConnectionAcceptor;
+import io.servicetalk.transport.api.ConnectionContext;
+import io.servicetalk.transport.api.IoExecutor;
+import io.servicetalk.transport.netty.internal.*;
+
+import static io.servicetalk.concurrent.api.Single.succeeded;
+
+
+
+public final class Server {
+
+
+    public static void main(String[] args) throws Exception {
+
+        /**
+         * Disable  AsyncContext
+         */
+        AsyncContext.disable();
```

Comment:
For comments within Java methods, use `// comments like this` or `/* comments like this */`, not `/** javadoc like this */`.  The double asterisk is for javadoc, which can be applied at the class level or member (field or method) level, but not within a method.